src/main/webapp/query-control.js

```diff
@@ -14,9 +14,20 @@ function registerServiceWorker() {
   }
 }
 
+// Sets the value of the color input. Checks to see if a color is in 
+// the cache already, and returns a nice blue if it isn't.
+function getColor() {
+  if (localStorage.getItem('color') !== null) {
+    document.getElementById('setColor').value = localStorage.getItem('color');
+  } else {
+    document.getElementById('setColor').value = '#0071bd';
```

Comment:
localStorage color could be given initial value here

Reply:
true! fixed